.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,11 @@ This template is for bug reports. If you are reporting a bug, please continue on
 Note that leaving sections blank will make it difficult for us to troubleshoot and we may have to close the issue.
 -->
 
-**What version of ESLint are you using?**
+**Tell us about your environment**
+
+* **ESLint Version:**
+* **Node Version:**
+* **npm Version:**
```

Comment:
The downside is higher bar of entry. If we ask for it in the template but don't insist on people providing that information if they skipped it, I'm fine with it. But requiring 100% of people to provide it when it might only be relevant in 2% of cases is an overkill.